### PR TITLE
reorg security service naming

### PR DESCRIPTION
**Pull request by finp**
## Motivation

making the MSS name work

## What
edit attributes, modules  and nav.adoc 
## Progress:

- [*] bring both services together
- [*] Change names
- [*] make nav work

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -77,16 +77,16 @@ include::{partialsdir}/attributes.adoc[]
...
-*** xref:app-security.adoc[{app-security-service}]
-**** xref:app-security.adoc#introduction[Introduction]
-**** xref:app-security.adoc#setup[Setting Up]
-**** xref:app-security.adoc#monitoring[Monitoring App Security]
-*** xref:device-security.adoc[{device-security-service}]
-**** xref:device-security.adoc#introduction[Introduction]
-**** xref:device-security.adoc#setup[Setting Up]
-**** xref:device-security.adoc#device-trust[Performing Device Trust Checks]
-**** xref:device-security.adoc#cert-pinning[Certificate Pinning]
+** xref:security.adoc[Mobile Security]
```
Comment (laurafitzgerald) [inline modules/ROOT/nav.adoc]: Screenshot 2019-07-03 at 09 38 43

Both the root of this nav and the first option are `Mobile Security` now.

File: modules/ROOT/pages/security.adoc [hunk 2]
```diff
@@ -2,23 +2,15 @@
 include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 
-= Security Services
+= Making your mobile app secure
```
Comment (laurafitzgerald) [inline modules/ROOT/pages/security.adoc]: I don't know what the syntax should be but some of the headings in this page use lower case and some use uppercase. Just an observation. Not sure it's relevant to this pr.

File: modules/ROOT/pages/security.adoc [hunk 3]
```diff
@@ -2,23 +2,15 @@
...
 
-There are two security services available:
+This guide does not address all mobile security topics. See <link> for an introduction to mobile security.
```
Comment (laurafitzgerald) [inline modules/ROOT/pages/security.adoc]: What link will go here?

File: modules/ROOT/pages/security.adoc [hunk 4]
```diff
@@ -2,23 +2,15 @@
...
+{org-name} helps you achieve security using:
 
+* {app-security-service} service - remotely disable specific app variants
```
Comment (laurafitzgerald) [inline modules/ROOT/pages/security.adoc]: Again, some discrepancy in upper and lower case.

File: modules/ROOT/pages/security.adoc [hunk 5]
```diff
@@ -2,23 +2,15 @@
...
+* {device-security-service} - for example, disallow running app on rooted devices
```
Comment (laurafitzgerald) [inline modules/ROOT/pages/security.adoc]: Could we link to the Device Security Section from here? And vice versa in a similar heading in the other section?

**Comment by laurafitzgerald** [comment]
Hi @finp thanks for the draft. I left some general comments inline but I wanted to ask about larger goal of the pr. I can see that one is to `bring both services together`. 
I believe from speaking to you that we aim to do that in the future but they are still both seperate services at the moment. I'm conscious theres some discussion already around this so what I've asked here is not final by any means.
If the goal now is to have both sections in one, then perhaps we can remove some of the additional nav items. 
If we are not doing that yet, but at some point in the future, I think we can still make naming improvements here.

File: modules/ROOT/pages/_partials/app-security/proc-calling-the-app-security-clientInit.adoc [hunk 6]
```diff
@@ -1,13 +1,8 @@
-= Using the {app-security-service} Service
+= Coding your mobile app to use the {app-security-service} service
```
Comment (damienomurchu) [inline modules/ROOT/pages/_partials/app-security/proc-calling-the-app-security-clientInit.adoc]:
```suggestion
= Implementing the {app-security-service} service in your mobile app
```

File: modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc [hunk 7]
```diff
@@ -1,21 +1,23 @@
-= Accessing the console
+= Controlling mobile apps using the Mobile Security Console
```
Comment (damienomurchu) [inline modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc]:
```
= Managing mobile apps using the Mobile Security Console
```

File: modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc [hunk 8]
```diff
@@ -27,7 +29,7 @@ This view provides statistics and information about each deployed version of an
 * *Disable on Startup*: Whether this application version is disabled on startup.
 * *Custom Disable Message*: A custom message that is displayed when this version of the mobile app is disabled.
 
-== Enabling and Disabling an application version
+== Enabling and disabling a mobile app
```
Comment (damienomurchu) [inline modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc]: I think it's important to note that there may be multiple versions of a mobile app, and the MSS allows you to enable/ disable these individually. In that regard I would perhaps suggest:

```suggestion
Enabling and disabling mobile app versions
```

File: modules/ROOT/pages/_partials/attributes.adoc [hunk 9]
```diff
@@ -37,8 +37,8 @@
 :mobilecicd-ui: Jenkins UI
 
 // Device Security
-:device-security-service: Device Security
-:app-security-service: App Security
+:device-security-service: Device Checks
+:app-security-service: Mobile Security
```
Comment (damienomurchu) [inline modules/ROOT/pages/_partials/attributes.adoc]: `Mobile Security` seems a bit vague to me

/2cents

File: modules/ROOT/pages/device-security.adoc [hunk 10]
```diff
@@ -11,8 +11,9 @@ include::_partials/attributes.adoc[]
 :context: security
 ifdef::context[:parent-context: {context}]
 
-[id='a-collection-of-modules']
-= Using the {device-security-service} Service
+[id='device-checks']
+= Adding {device-security-service} to your mobile app
```
Comment (damienomurchu) [inline modules/ROOT/pages/device-security.adoc]:
```suggestion
= Implementing {device-security-service} in your mobile app
```

File: modules/ROOT/pages/security.adoc (hunk 2 above)
Comment (damienomurchu) [inline modules/ROOT/pages/security.adoc]:
```suggestion
Securing your mobile app
```

File: modules/ROOT/pages/_partials/attributes.adoc (hunk 9 above)
Comment (finp) [reply modules/ROOT/pages/_partials/attributes.adoc]: prob not a great place to review that, cos in the docs it typically appears as 'Mobile Security Service'
and that name comes from Product/Eng, so no option to change it...
in fact, changing it to this value is the whole purpose of this PR.

**Comment by damienomurchu** [comment]
Changes look good @finp :+1:  I see you've already starting actioning some of the feedback items I left in comments - let me know when you're finished & I will re-review and approve

File: modules/ROOT/pages/security.adoc (hunk 2 above)
Comment (finp) [reply modules/ROOT/pages/security.adoc]: need to follow up in nav.adoc

File: modules/ROOT/pages/_partials/attributes.adoc (hunk 9 above)
Comment (damienomurchu) [reply modules/ROOT/pages/_partials/attributes.adoc]: Ah got ya - for some reason I skimmed quickly over this area and thought it was just an internal heading name we were using :) 

Anyway, agreed :) :+1:

File: modules/ROOT/nav.adoc [hunk 11]
```diff
@@ -77,16 +77,15 @@ include::{partialsdir}/attributes.adoc[]
...
-*** xref:app-security.adoc[{app-security-service}]
-**** xref:app-security.adoc#introduction[Introduction]
-**** xref:app-security.adoc#setup[Setting Up]
-**** xref:app-security.adoc#monitoring[Monitoring App Security]
-*** xref:device-security.adoc[{device-security-service}]
-**** xref:device-security.adoc#introduction[Introduction]
-**** xref:device-security.adoc#setup[Setting Up]
-**** xref:device-security.adoc#device-trust[Performing Device Trust Checks]
-**** xref:device-security.adoc#cert-pinning[Certificate Pinning]
+** xref:security.adoc[Secure your app]
```
Comment (damienomurchu) [inline modules/ROOT/nav.adoc]: Missed this on my first pass - does it make sense to change this as all other items in this list are actual mobile services, and this change stands out, ie:

Screenshot from 2019-07-19 12-29-02

Comment (finp) [reply modules/ROOT/nav.adoc]: good point, but will address later

**Pull request merged**